HousingSearchApi.Tests/V2/E2ETests/BaseSearchTests.cs

```diff
@@ -7,32 +7,34 @@
 using System.Threading.Tasks;
 using HousingSearchApi.Tests.V2.E2ETests.Fixtures;
 using Microsoft.Extensions.Logging;
-using Xunit;
 
 namespace HousingSearchApi.Tests.V2.E2ETests;
 
+using Nest;
 using Xunit;
 
+
 [CollectionDefinition("V2.E2ETests Collection", DisableParallelization = true)]
-public class V2E2ETestsCollection
+public class V2E2ETestsCollection : ICollectionFixture<CombinedFixture>
```

Comment:
This was great so that the setup only runs once for all tests which makes them faster and more reliable